### DISCUSSION TITLE
Vue 3 update is not clear in Read.me

### DISCUSSION THREAD

**Discussion by yogeshgalav** [shentao/vue-multiselect#1693] [General]
From first glance at Homepage or github page/ Read.me 
It's not clear if vue 3 support is available or not.
Which two components we need to remove or swap?
The blog link to suade.org is also not working.

If vue 3 support is available, then can we update website home page and github page with message,
"Vue 3 support is now available."

**Comment by mattelen** [comment]
Good point. The blog link is already working, and with the merge of https://github.com/shentao/vue-multiselect/pull/1815 we've made it a bit clearer that this works with Vue 3. Thanks for bringing this up
